setup.py

```diff
@@ -40,13 +40,12 @@
 INSTALL_REQUIRES = ['numpy >= 1.10.1',
                     'pandas >= 0.16.0',
                     'pytz',
-                    'requests',
-                    'six',
-                    ]
-TESTS_REQUIRE = ['pytest', 'pytest-cov', 'pytest-mock', 'nose']
+                    'requests']
+TESTS_REQUIRE = ['nose', 'pytest', 'pytest-cov', 'pytest-mock',
+                 'pytest-timeout']
 EXTRAS_REQUIRE = {
     'optional': ['scipy', 'tables', 'numba', 'siphon', 'netcdf4',
-                 'ephem', 'cython', 'pvfactors'],
+                 'ephem', 'cython', 'pvfactors == 1.0.1'],
```

Comment:
I'd rather not pin the dependency our setup.py file.

Reply:
Is this a minimum? It is pinned in the build environments?